configure

```diff
@@ -22,8 +22,27 @@ fi
 CC="${CC:-cc}"
 PKG_CONFIG="${PKG_CONFIG:-pkg-config}"
 
-# set hostonly by default
-configprofile="${configprofile:-hostonly}"
+# set configprofile based on ID_LIKE from os-release
+if [[ -f /etc/os-release ]]; then
+    . /etc/os-release
+elif [[ -f /usr/lib/os-release ]]; then
+    . /usr/lib/os-release
+fi
+
+case "$ID$ID_LIKE" in
+    fedora*)
+        configprofile=fedora
+        ;;
+    debian*)
+        configprofile=debian
+        ;;
+    *suse)
+        configprofile=opensuse
+        ;;
+    *)
+        configprofile=hostonly
+        ;;
+esac
```

Comment:
`$ID$ID_LIKE` would be `ubuntudebian` on Ubuntu.

`ID_LIKE` is a space separated list. So I suggest following to search for matching IDs (plus alphabetical order):

```suggestion
case " $ID $ID_LIKE " in
    *" debian "*)
        configprofile=debian
        ;;
    *" fedora "*)
        configprofile=fedora
        ;;
    *" suse "*)
        configprofile=opensuse
        ;;
    *)
        configprofile=hostonly
        ;;
esac
```

I grabbed samples from https://github.com/chef/os_release to check that the distributions set ID or ID_LIKE to those values.